community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -567,6 +567,10 @@
     public static final Setting<String> procedure_unrestricted =
             setting( "dbms.security.procedures.unrestricted", Settings.STRING, "" );
 
+    @Description("If dbms.killQueries give a verbose output, with information about whitch querys where not found." )
```

Comment:
The intent is to not change old behavior in a patch release for 3.2. In the forward merge, this config should be flipped to support the verbose as the default.